### DISCUSSION TITLE
How to work around with found conflicting index store units?

### DISCUSSION THREAD

**Discussion by yuya-h-29** [peripheryapp/periphery#612] [Q&A] [answered]
Hello, thank you for providing an amazing library. I've been using this for a few months and it helps me a lot when refactoring my code. Until a week ago, it was perfectly fine and I had no issue with `periphery scan`  command. However, now I get the error saying 

> Found conflicting index store units for 'A PATH FOR A SPECIFIC .swift FILE' in module 'myAPP'. If you passed the '--index-store-path' option, ensure that Xcode is not open with a project that may write to this index store while Periphery is running. 

I tried the things below, but I still get the same error, so I want someone to point me in the right direction. 

#### Things I tried

- remove the .swift file I get in the error and rerun periphery scan.
    -> gets the same error with a new path for a `.swift` file
- clean build and run scan command
    -> gets the same `Found conflicting index store` error
- close Xcode and run scan command in terminal
    -> gets the same `Found conflicting index store` error
- run `periphery scan --setup` command and start all over
    -> gets the same `Found conflicting index store` error
- remove DerivedData and run scan command
    -> gets the same `Found conflicting index store` error
    
When I encounter with the `Found conflicting index store units` error, what and how to fix the issue?
Also, if you know why this happens, I want to know what I have possibly done to cause this error.

**Comment by amleszk** [comment, marked as answer]
Hello I have the same issue and tried all the steps you listed also, looks like resolved it by clearing out derived data periphery will store in `/Users/[[USER]]/Library/Caches/com.github.peripheryapp`

**Comment by yuya-h-29** [comment]
@amleszk 
Hello and thank you for your information! Just figured out your reply and tried deleting derived data and now it works in my terminal!!
You made my day! Arigato :)
